src/patternfly/components/ProgressStepper/progress-stepper.scss

```diff
@@ -206,6 +207,10 @@
   &.pf-m-info {
     --pf-c-progress-stepper__step-icon--Color: var(--pf-global--info-color--100);
   }
+
+  &:last-child .pf-c-progress-stepper__step-main {
```

Comment:
Could this also be written this way with less specificity?

```
.pf-c-progress-stepper__step:last-child {
    --pf-c-progress-stepper__step-main--MarginBottom: var(--pf-c-progress-stepper__step--last-child__step-main--MarginBottom);
}
```

We also often don't create a var for something like this that we wouldn't expect to be customized, so your declaration could also be `--pf-c-progress-stepper__step-main--MarginBottom: 0;`, and remove the var. But the way you've done it is fine, too.

Reply:
I removed the variable, but you can't remove specificity by removing step-main because then you aren't targeting what is causing the extra space.

Reply:
Would this not work? https://codesandbox.io/s/dank-morning-bc5r4?file=/style.css

Reply:
Ah, yes, if you set the variable instead of margin-bottom directly. 🤪